mesonbuild/utils/universal.py

```diff
@@ -30,6 +30,7 @@
 
 if T.TYPE_CHECKING:
     from typing_extensions import Literal, Protocol
+    from typing import TypedDict
```

Comment:
needs to be typing_extensions, it's not in typing until 3.8, and we still support 3.7